### DISCUSSION TITLE
Django-select2 doesn't work on inlineformset_factory?

### DISCUSSION THREAD

**Discussion by bentraje** [codingjoe/django-select2#68] [Q&A]
Hi, 

Thanks for sharing the project.

The select2 works perfectly on just a standalone model form but if its with inlineform, it doesn't work expected. 
Is there an additional code (not coverred in the documention) for using  Django-select2 for inlineform_factory?

**Comment by codingjoe** [comment]
Hi @bentraje,

Thank you for reaching out. Would you mind sharing your code, to help me debug your problem better?
It should work, I have used this packages in all kinds of scenarios in the past. Therefore, I am positive, that we can find a solution.

Best,
Joe
